app/models.py

```diff
@@ -0,0 +1,25 @@
+from sqlalchemy import Column, Integer, String, ForeignKey, DateTime, Float
+from sqlalchemy.orm import relationship
+
+from database import Base
+
+
+class DBCity(Base):
+    __tablename__ = "city"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String(50), nullable=False)
```

Comment:
The length of the 'name' column in the 'city' table is set to 50 characters. This might not be sufficient for some city names especially when considering cities from around the world. It's a good practice to set a reasonable maximum length that can accommodate all possible values. Consider increasing the length to a larger value.